src/components/data-management/Header.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Row, Typography, Space, Button, Dropdown, Col,
+} from 'antd';
+import {
+  updateProject,
+} from '../../redux/actions/projects'; import DownloadData from './DownloadData';
+
+const { Title, Text, Paragraph } = Typography;
+
+const Header = (props) => {
+  const {
+    activeProjectUuid, createMetadataColumn, isAddingMetadata, canLaunchAnalysis,
+    setUploadModalVisible, openAnalysisModal,
+  } = props;
+  const dispatch = useDispatch();
+  const activeProject = useSelector((state) => state.projects[activeProjectUuid]) || false;
+  const projects = useSelector((state) => state.projects);
+  const changeDescription = (description) => {
+    dispatch(updateProject(activeProjectUuid, { description }));
+  };
+  return (
```

Comment:
instead of checking in every bit in here if `projects.ids.length === 0`, I would add an if statement for it before this return, then if the if is true, we can just not load anything (just return `<></>`), if it is false, then do the logic you have in here, but remove all these `activeProject?.samples?.length === 0` checks